conjure-java-core/src/integrationInput/java/com/palantir/product/EmptyUnionTypeExample.java

```diff
@@ -31,6 +32,13 @@ private Base getValue() {
         return value;
     }
 
+    public static EmptyUnionTypeExample unknown(String type, Object value) {
+        switch (Preconditions.checkNotNull(type, "Type is required")) {
+            default:
+                return new EmptyUnionTypeExample(new UnknownWrapper(type, Collections.singletonMap(type, value)));
```

Comment:
a little wonky, but probably no need to special case this one as long as it compiles :-)